showkase-processor-testing/src/androidTest/java/com/airbnb/android/showkase_processor_testing/ShowkaseBrowserTest.kt

```diff
@@ -24,12 +24,20 @@ class ShowcaseBrowserTest {
     // android.app.Activity in the AndroidManifest.xml
     @get:Rule
     val composeTestRule =
-        AndroidComposeTestRule<ShowkaseBrowserActivity>(
-            ActivityScenarioRule(
+        AndroidComposeTestRule(
+            activityRule = ActivityScenarioRule<ShowkaseBrowserActivity>(
                 createShowkaseBrowserIntent(
                     InstrumentationRegistry.getInstrumentation().targetContext
                 )
-            )
+            ),
+            activityProvider = { rule ->
+                var activity: ShowkaseBrowserActivity? = null
+                rule.scenario.onActivity { activity = it }
+                if (activity == null) {
+                    throw IllegalStateException("Activity was not set in the ActivityScenarioRule!")
```

Comment:
a bit shorter
```suggestion
                    error("Activity was not set in the ActivityScenarioRule!")
```

or
```
return activity ?: error("Activity was not set in the ActivityScenarioRule!")
```
even shorter and removes the need to `!!` which is ideal